src/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/function-component-definition */
+import React from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { ROUTES } from 'Data/constants';
+
+import { ReactComponent as ClockDark } from 'Assets/clockDark.svg';
+
+import './index.scss';
+
+const Sidebar = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const items = [
+    {
+      label: 'timer',
+      icon: 'timerIcon',
+      path: ROUTES.timer,
+    },
+    {
+      label: 'report',
+      icon: 'reportIcon',
+      path: ROUTES.report,
+    },
+  ];
+  return (
+    <div className="sidebar">
+      <ClockDark className="sidebar__clock" />
+      {items.map((item) => (
+        <button type="submit" className={item.path === location.pathname ? 'sidebar__button active' : 'sidebar__button inactive'} onClick={() => navigate(item.path)}>
```

Comment:
Que te parece si dividimos este bloque en nuevas lineas? Asi no queda tan largo.
```suggestion
        <button 
          type="submit" 
          className={item.path === location.pathname ? 'sidebar__button--active' : 'sidebar__button--inactive'} 
          onClick={() => navigate(item.path)}
        >
```

Tambien, si podes seguir `BEM` mejor. Para el caso de `active` e `inactive`.

```
.sidebar {
  ...
  
 &__button {
  ...
 
  &--active {
    ...
  }
  
  &--inactive {
    ...
  }
 }
}